CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Changes
 
 - [graph] Updated the Graph model and add Layoutable interface [#175](https://github.com/eclipse-glsp/glsp-server/pull/175) - Contributed on behalf of STMicroelectronics
+- [operation] Rework `OperationHandler` to provide an optional command instead of direct execution to allow more execution control [#187](https://github.com/eclipse-glsp/glsp-server/pull/187)
```

Comment:
Should probably be declared as breaking change because the `OperationHandler` interface is no longer compatible.
Also it would be nice to mention that we deprecated the `AbstractXYZ` implementations and removed the old `BasicXYZOperationHandler`s (which where announced to be removed before 1.0.0, so we even kept them longer as necessary)